umap/umap_.py

```diff
@@ -93,11 +93,11 @@ def smooth_knn_dist(
 
     Returns
     -------
-    knn_dist: array of shape (n_samples,)
-        The distance to kth nearest neighbor, as suitably approximated.
+    result: array of shape(n_samples)
```

Comment:
I think this should rather be called `sigmas` to be in-line with https://github.com/lmcinnes/umap/blob/f3f2dac5517d065bf799c351da0d3f3d2203a02b/umap/umap_.py#L326